astropy/io/fits/hdu/table.py

```diff
@@ -702,6 +686,17 @@ class TableHDU(_TableBaseHDU):
         r'(?P<code>[ADEFIJ])(?P<width>\d+)(?:\.(?P<prec>\d+))?')
 
     def __init__(self, data=None, header=None, name=None):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
```

Comment:
The docstring was inherited so it was used, but ok I will tidy up the docstrings ;)
http://docs.astropy.org/en/latest/io/fits/api/tables.html#astropy.io.fits.BinTableHDU